toolchest_client/tools/api.py

```diff
@@ -1201,10 +1200,9 @@ def add_database(database_path, tool, database_name, database_primary_name=None,
     :param database_path: Path or list of paths (local or S3) to be passed in as inputs.
     :param tool: Toolchest tool with which you use the database (e.g. toolchest.tools.Kraken2).
     :param database_name: Name of the new database.
-    :param database_primary_name: Name or path of the file to use as the primary database file
-        (i.e., what you would pass into the command line as the database), if uploading multiple
-        files. If unspecified, assumes that the *directory* of files is what will be passed in
-        as the database.
+    :param database_primary_name: Name or path of the file/prefix that will be passed as the database,
+        just like the command line call. Use `database_primary_name=None` to pass the entire *directory*
```

Comment:
"path" might be a bit confusing here since a command might take `/path/to/prefix` and we only want `prefix` passed in. Similarly I think there may be a better way to say "just like the command line call".